layout/_third-party/math/mathjax.swig

```diff
@@ -1,53 +1,44 @@
-{%- set mhchem_uri = theme.vendors.mhchem or '//cdn.jsdelivr.net/npm/mathjax-mhchem@3' %}
-<script type="text/x-mathjax-config">
-  {%- if theme.math.mathjax.mhchem %}
-    MathJax.Ajax.config.path['mhchem'] = '{{ mhchem_uri }}';
-  {%- endif %}
+{%- set mathjax_uri = theme.vendors.mathjax or '//cdn.jsdelivr.net/npm/mathjax@3.0.1/es5/tex-mml-chtml.js' %}
 
-  MathJax.Hub.Config({
-    tex2jax: {
-      inlineMath: [ ['$', '$'], ['\\(', '\\)'] ],
-      processEscapes: true,
-      skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
-    },
-    TeX: {
+<script>
+  if (typeof MathJax === 'undefined') {
+    window.MathJax = {
       {%- if theme.math.mathjax.mhchem %}
-        extensions: ['[mhchem]/mhchem.js'],
+        loader: {
+          load: ['[tex]/mhchem']
+        },
       {%- endif %}
-      equationNumbers: {
-        autoNumber: 'AMS'
-      }
-    }
-  });
-
-  MathJax.Hub.Register.StartupHook('TeX Jax Ready', function() {
-    MathJax.InputJax.TeX.prefilterHooks.Add(function(data) {
-      if (data.display) {
-        var next = data.script.nextSibling;
-        while (next && next.nodeName.toLowerCase() === '#text') {
-          next = next.nextSibling;
-        }
-        if (next && next.nodeName.toLowerCase() === 'br') {
-          next.parentNode.removeChild(next);
+      tex: {
+        inlineMath: {'[+]': [['$', '$']]},
+        {%- if theme.math.mathjax.mhchem %}
+          packages: {'[+]': ['mhchem']},
+        {%- endif %}
+        tags: 'ams'
+      },
+      options: {
```

Comment:
@reuixiy Can you confirm the role of this `options`? I removed it and found no issues with rendering. Also, I found the documentation: http://docs.mathjax.org/en/v3.0-latest/upgrading/v2.html#changes-in-the-mathjax-api

Reply:
For hexo-renderer-kramed, I found that this plugin is no longer maintained (the last release was 3 years ago). Maybe we need to find an alternative to it.

Reply:
Yep :)

Reply:
Since kramed itself is no longer maintained, using [hexo-renderer-pandoc](https://www.npmjs.com/package/hexo-renderer-pandoc) or hexo-renderer-marked instead would be a good choice. theme-next.org has been updated: https://github.com/theme-next/theme-next.org/commit/520c38d300f234f99928f2b7a445a208f7ab24de now you can safely remove this option.